nlmod/gwf/surface_water.py

```diff
@@ -1024,8 +1024,10 @@ def add_season_timeseries(
     package,
     summer_months=(4, 5, 6, 7, 8, 9),
     filename="season.ts",
-    seasons=("winter", "summer"),
+    seasons=None,
```

Comment:
Why set seasons to None? The keyword summer_months sort of suggests winter/summer being the only possible two seasons?

Reply:
I changed this as a tuple gave a bug, `seasons` (`time_series_namerecord`) somehow needed to be a list, and it is not allowed to to add a list as an optional arggument. Therefore when `seasons` is None, I set it to `seasons = ["winter", "summer"]`. Maybe better to add two arguments, called `winter_name` and `summer_name`?